iocBoot/iocEK9KTest/st.tpl

```diff
@@ -0,0 +1,21 @@
+#!../../bin/$ARCH$/ek9000Test
```

Comment:
Not worth changing, but for future reference you could also use `string.Template()` and then use standard-ish template syntax with `${EPICS_HOST_ARCH}` -style variable references

Only real downside is that then regular `$` would need to be escaped as `$$`

ref: https://docs.python.org/3/library/string.html#template-strings

Reply:
Python really has everything doesn't it?